client/k8s/client.go

```diff
@@ -12,7 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-package client
+package k8s
```

Comment:
K8s. Client is better than K8S. k8sClient
```
type K8sClient struct {
	client *kubernetes.Clientset
}
```

Reply:
good reviews